packages/sdk-vue/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@telegram-apps/sdk-vue",
+  "version": "2.0.0",
+  "description": "Vue.js bindings for Mini Apps client SDK.",
+  "author": "Vladislav Kibenko <wolfram.deus@gmail.com>",
```

Comment:
Type your own information here if you want to

Reply:
Ok, I suppose, it would be better if in future all vue-sdk related questions will be addressed to me (hope there won't be any :D)